src/PresentationalComponents/Email/Email.js

```diff
@@ -23,21 +24,24 @@ import config from '../../config.json';
 import { emailPreferences, register } from '../../store';
 import { getEmailSchema, saveEmailValues } from '../../actions';
 
-const FormButtons = ({ submitting, pristine, onCancel }) => (
-    <div>
-        <Button
-            type="submit"
-            isDisabled={ submitting || pristine }
-            style={ { marginRight: 16 } }
-            variant="primary">Save</Button>
-        <Button
-            variant="link"
-            isDisabled={ pristine }
-            onClick={ onCancel }>
+const FormButtons = ({ pristine, dirtyFieldsSinceLastSubmit, submitSucceeded, onCancel }) => {
+    const noChanges = isEmpty(dirtyFieldsSinceLastSubmit) || !submitSucceeded && pristine;
+    return (
+        <div>
+            <Button
+                type="submit"
+                isDisabled={ noChanges }
+                style={ { marginRight: 16 } }
```

Comment:
Can you write this into class and CSS selector with `margin-rights: var(--pf-global--gutter--md)`